rpm/sailfish-components-webview.spec

```diff
@@ -66,21 +66,30 @@ Requires:  %{name} = %{version}
 %{summary}.
 
 %package devel
-Summary:    Sailfish OS WebEngine development files
+Summary:    Sailfish WebEngine development files
 Requires:   %{name} = %{version}-%{release}
 
 %description devel
 Development package which provides libsailfishwebengine
 
 %package tests
-Summary:    Sailfish OS WebView / WebEngine tests
+Summary:    Sailfish WebView / WebEngine tests
```

Comment:
If you want do some additional cleanup, removing "Group:" from the spec would be a nice plus. Not mandatory though.

See:
https://sailfishos.org/wiki/Software_Packaging

Reply:
Have also done so.